src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,8 +1,17 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
 public class UserRequestDto {
+    @Email
     private String email;
+    @NotNull
```

Comment:
checkstyle: Don’t repeat checks for fields: you must ensure email field not to be null in your custom Validator, so there is no need to put @NotNull for that field as well in Dto class